### PR TITLE
Feature/cypress test case enhancement treemap

### PR DESCRIPTION
**Pull request by nidhisinghai**
Written Cypress Test Cases for Enhancement on TreeMap
-Add/Delete Multiple Parent Field
-Color Theme option for Multicolor on Multiple Parent

### CODE REVIEW

File: dashboards-observability/.cypress/integration/1_event_analytics.spec.js [hunk 1]
```diff
@@ -911,3 +911,74 @@ describe('Renders heatmap chart for Chart Style', () => {
...
+
+      it('Renders Tree Map and Check Add/Delete Parent', () => {
+        querySearch(TEST_QUERIES[7].query, TEST_QUERIES[7].dateRangeDOM);
+        cy.get('[data-test-subj="configPane__vizTypeSelector"] [data-test-subj="comboBoxInput"]').type('Tree Map').type('{enter}');
+        cy.get('.euiButton__text').contains('+ Add Parent').click();
+        cy.get('.euiComboBoxPlaceholder').contains('Select a field').should('exist');
+        cy.get('.euiButton__text').contains('+ Add Parent').click();
+        cy.get('.euiComboBoxPlaceholder').eq(1).contains('Select a field').should('exist');
+        cy.get('.euiFormRow__fieldWrapper .euiFlexItem').eq(1).click();
+        // cy.get('.euiComboBoxPlaceholder').contains('Select a field').should('not.exist');
```
Comment (abasatwar) [inline dashboards-observability/.cypress/integration/1_event_analytics.spec.js:956]: remove commented code

File: dashboards-observability/.cypress/integration/1_event_analytics.spec.js [hunk 2]
```diff
@@ -911,3 +911,74 @@ describe('Renders heatmap chart for Chart Style', () => {
...
    });
 });
+
+describe('Renders Tree Map for Parent Fields ', () => {
+  beforeEach(() => {
+    landOnEventVisualizations();
+  });
+
+  it('Renders Tree Map and Add Single Parent', () => {
+    querySearch(TEST_QUERIES[7].query, TEST_QUERIES[7].dateRangeDOM);
```
Comment (abasatwar) [inline dashboards-observability/.cypress/integration/1_event_analytics.spec.js:921]: For single parent test case can common for below test cases.

**Review by abasatwar — CHANGES_REQUESTED** [review]

File: dashboards-observability/.cypress/integration/1_event_analytics.spec.js [hunk 3]
```diff
@@ -911,3 +911,74 @@ describe('Renders heatmap chart for Chart Style', () => {
...
+    cy.get('[data-test-subj="configPane__vizTypeSelector"] [data-test-subj="comboBoxInput"]').type('Tree Map').type('{enter}');
+    cy.get('.euiButton__text').contains('+ Add Parent').click();
+    cy.get('.euiComboBoxPlaceholder').contains('Select a field').should('exist');
+    cy.get('.euiFormControlLayoutIcons [data-test-subj ="comboBoxToggleListButton"]').eq(3).click();
+    cy.get('.euiComboBoxOption__content').eq(2).click();
+    cy.get('.euiButton__text').contains('Preview').click();
+    });
+
+    it('Renders Tree Map and Add Multiple Parent', () => {
```
[the remark this answers is not in the document]
Comment (nidhisinghai) [reply dashboards-observability/.cypress/integration/1_event_analytics.spec.js:930]: Done

File: dashboards-observability/.cypress/integration/1_event_analytics.spec.js (hunk 1 above)
Comment (nidhisinghai) [reply dashboards-observability/.cypress/integration/1_event_analytics.spec.js:956]: Done

File: dashboards-observability/.cypress/integration/1_event_analytics.spec.js (hunk 2 above)
Comment (nidhisinghai) [reply dashboards-observability/.cypress/integration/1_event_analytics.spec.js:921]: removed this test case

**Review by abasatwar — APPROVED** [review]

**Comment by nidhisinghai** [comment]
new separate files are created under visualizationschart folder

**Pull request closed without merging**